Resources/config/default_settings.yml

```diff
@@ -1,11 +1,7 @@
-parameters:
-    ez_recommendation.default.yoochoose.customer_id: ~
-    ez_recommendation.default.yoochoose.license_key: ~
-    ez_recommendation.default.server_uri: ~
-    ez_recommendation.default.recommender.included_content_types: [article, blog_post, place, product]
-
-    ez_recommendation.api_endpoint: 'https://admin.yoochoose.net'
-    ez_recommendation.recommender.api_endpoint: 'http://event.yoochoose.net'
-    ez_recommendation.recommender.consume_timeout: 20
-    ez_recommendation.tracking.api_endpoint: 'http://event.yoochoose.net'
-    ez_recommendation.tracking.script_url: 'cdn.yoochoose.net/yct.js'
+api_endpoint: 'https://admin.yoochoose.net'
+recommender:
+    api_endpoint: 'http://event.yoochoose.net'
+    consume_timeout: 20
+tracking:
+    api_endpoint: 'http://event.yoochoose.net'
+    script_url: 'cdn.yoochoose.net/yct.js'
```

Comment:
`default_settings.yml` is not supposed to store semantic config, but *internal* default values (like what was in the file before, e.g. `ez_recommendation.default.server_uri`), that would have resulted by the semantic config processing.
Check [what we have in ezpublish-kernel](https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Bundle/EzPublishCoreBundle/Resources/config/default_settings.yml) for instance.

This is done to ensure we have correct default values.